e2e/optimism_test.go

```diff
@@ -238,6 +259,24 @@ func testOptimismGenericCommitment(t *testing.T, v2Enabled bool) {
 	// verify
 	optimism.sequencer.ActL2PipelineFull(ot)
 	optimism.ActL1Finalized(ot)
+}
+
+func TestOptimismGenericCommitmentMigration(t *testing.T) {
+	// initialize with v2 = true, so that the necessary components are constructed
+	proxyTS, shutDown := testutils.CreateTestSuite(testutils.GetBackend(), true)
+	defer shutDown()
+
+	// turn off v2 dispersal, so that we exercise v1, i.e. "pre migration"
+	proxyTS.Server.SetDisperseV2(false)
+
+	ot := actions.NewDefaultTesting(t)
```

Comment:
How are we actually asserting that proxy is dual processing both V1 and V2 if we're only verifying dispersal/retrieval based on a generalized metrics vector. Would there be merit in extending the telemetry to also support a versioning field?

Reply:
I'd argue for guaranteed migration test coverage its absolutely necessary

Reply:
I agree, the assertions here just aren't strong enough.

I think it makes sense to create a follow-up PR, I [created an issue](https://linear.app/eigenlabs/issue/DAINT-381/strengthen-proxy-test-assertions-for-v1-vs-v2-operation)